backend/src/main/kotlin/app/ehrenamtskarte/backend/cards/webservice/schema/CardMutationService.kt

```diff
@@ -147,6 +147,7 @@ class CardMutationService {
 
         val userEntitlements = transaction { UserEntitlementsRepository.findUserEntitlements(userHash.toByteArray()) }
         if (userEntitlements == null || userEntitlements.revoked || userEntitlements.endDate.isBefore(LocalDate.now())) {
+            logger.info("${context.remoteIp} ${userEntitlements?.userHash} failed to create a new card")
```

Comment:
Hm as @seluianova described in the issue the rate limiting (and logging) should apply if no cardHash can be found
so
`if (userEntitlements == null)` and not if expired or revoked.
I don't think we should also log and possibly ban user that were revoked or expired.

When we optimize our user messages we will even differ between not found or expired and will throw different exceptions. At least i think we should do this.
So we will need a separate `== null` condition